src/sentry/models/sentryappwebhookerror.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import absolute_import
+
+from django.db import models
+from django.utils import timezone
+
+from sentry.db.models import EncryptedJsonField, FlexibleForeignKey, Model
+
+
+class SentryAppWebhookError(Model):
+    __core__ = False
+
+    date_added = models.DateTimeField(default=timezone.now)
```

Comment:
@nolachen I think we need `db_index=True` here so we can query this efficiently.